FlatCache/FlatCache.swift

```diff
@@ -136,5 +136,9 @@ public final class FlatCache {
         return storage[key] as? T
     }
 
+    public func clearCache() {
```

Comment:
Kind of feel the name is weird from. the caller site. For now it looks like 
```swift
cache.clearCache()
```
May be we could short down to 
```swift
cache.clear()
```

Reply:
The listeners should be told that the cache that they're observing is being cleared (in my opinion at least).  In my PR #4 I do go through the listeners and notify the ones that are listening that an item is being removed from the cache.  I would say that removing a single item logic should come before the clearing the cache as that is essentially performing multiple "_removes_" which would update the listeners as well.  @rnystrom thoughts?